pkg/hooks/install.go

```diff
@@ -0,0 +1,69 @@
+package hooks
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/square/p2/Godeps/_workspace/src/github.com/Sirupsen/logrus"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/pods"
+	"github.com/square/p2/pkg/runit"
+	"github.com/square/p2/pkg/util"
+)
+
+// Populates the given directory with executor scripts for each launch script of
+// the given pod, which must be installed. Any orphaned executor scripts (from a
+// past install, but no longer present in this pod) will be cleaned out.
+func InstallHookScripts(dir string, hookPod *pods.Pod, manifest pods.Manifest, logger logging.Logger) error {
+	err := os.MkdirAll(dir, 0755)
+	if err != nil {
+		return err
+	}
+
+	rmPattern := filepath.Join(dir, fmt.Sprintf("%s__*", hookPod.Id))
```

Comment:
Still think this is worth leaving a refactoring note about changing the wildcard glob